Projects/App/Sources/Screens/ChatScreen/Chat.swift

```diff
@@ -46,9 +47,28 @@ extension Chat: Presentable {
 		bag += navigateCallbacker.onValue { navigationEvent in
 			switch navigationEvent {
 			case .offer:
-				viewController.present(
-					Offer(offerIDContainer: .stored, menu: Menu(title: nil, children: []))
-				)
+                client.fetch(query: GraphQL.LastQuoteOfMemberQuery()).onValue { data in
+                    guard let id = data.lastQuoteOfMember.asCompleteQuote?.id else {
+                        return
+                    }
+                    
+                    viewController.present(
```

Comment:
it would be great to pull navigation out of screen level - now that we have an offer module would it be possible to create a `ChatFlow` object that manages more of the navigation? Is this something we wanna commit to longterm - as in slowly moving `viewController.present` to a dedicated space so we don't run into cross dependency issues? ie we don't want Chat to import Offer?

Reply:
@tarikstafford yeah, get ya, `Chat` is a part of `App` now though, so lets do this later? this is gonna be obsolete quite soon anyways.